www/src/Gallery.jsx

```diff
@@ -26,9 +26,10 @@ export const EditorGallery = () => {
           onChange={handleChange}
           value={blockType}
         >
-          <Form.Radio value="html">Text</Form.Radio>
-          <Form.Radio value="video">Video</Form.Radio>
-          <Form.Radio value="problem">Problem</Form.Radio>
+          { blockTypes.map((e) => (
+            <Form.Radio value={e}> {e} </Form.Radio>
+          ))}
+
```

Comment:
blockTypes is not an array here and will trigger an error.
Something like the following code should work.
```
          { Object.values(blockTypes).map((e) => (
            <Form.Radio value={e}> {e} </Form.Radio>
          ))}
```